packages/nimble-components/src/.eslintrc.js

```diff
@@ -1,168 +1,8 @@
-const restrictedImportsPaths = () => [
-    {
-        name: '@microsoft/fast-foundation',
-        importNames: ['focusVisible'],
-        message:
-            'Please use focusVisible from src/utilities/style/focus instead.'
-    },
-    {
-        name: '@microsoft/fast-components',
-        message:
-            'It is not expected to leverage @microsoft/fast-components directly as they are coupled to FAST design tokens.'
-    }
-];
-
 module.exports = {
     root: true,
-    plugins: ['jsdoc'],
-    extends: [
-        '@ni/eslint-config-typescript',
-        '@ni/eslint-config-typescript/requiring-type-checking'
-    ],
+    extends: 'plugin:@ni-private/nimble/typescript',
```

Comment:
this will end up looking more like a normal configuration override / like using the ni shared configs instead of using a plugin

```suggestion
    extends: '@ni-private/eslint-config-nimble/typescript',
```